device-connectors/src/testflinger_device_connectors/devices/muxpi/muxpi.py

```diff
@@ -33,6 +35,11 @@
 logger = logging.getLogger(__name__)
 
 
+# should mirror `testflinger_agent.config.ATTACHMENTS_DIR`
+# [TODO] Merge both constants into testflinger.common
+ATTACHMENTS_DIR = "attachments"
```

Comment:
We actually have some of what we need in place to set up a testflinger_common right now, but is this something that really needs to be referenced by multiple projects? We should chat about how to structure the common pieces we import from testflinger_common and I can help get things rolling with that soon, but doesn't need to block this PR

Reply:
The agent needs this constant (already defines it) to know where to extract the attachments and now the device connector needs it (and redefines it) to know where to look for the attachments. If one changes so should the other so I would definitely say this is common to multiple projects.